language/testing-infra/diem-transactional-test-harness/tests/diem_test_harness/example.move

```diff
@@ -1,4 +1,5 @@
 //# init --addresses A=0x4777eb94491650dd3f095ce6f778acb6
+//#      --private-keys A=56a26140eb233750cd14fb168c3eb4bd0782b099cde626ec8aff7f3cceb6364f
```

Comment:
I was imagining we would just auto generate these for the test. How would I generate this when writing the test?

Reply:
1. There is a new tool/crate called `diem-keygen` that can help you with this. Run `cargo run -p diem-keygen`, then copy-paste the private key, auth key prefix and account address to your test. As I mentioned previously, I should have this better documented.
2. In case you are asking whether it's possible to let the test harness generate this for you so you don't even have to run the tool above, then the short answer is No. First, the account address is derived from the key pair, so if we want the private key to be auto-managed, then we must also auto-manage the address, which makes the Diem init command divergent from the Move one. Second, in order to call the account creation functions, you'll need to supply the auth key prefix. Having the key pair auto-generated/managed means you'll either need to implement some sort of text substitution mechanism (like what we had in functional tests) or add a magical account creation command. Neither seem desirable to me since we value explicitness.